Manifests/ManagedPreferencesApplications/com.almenscorner.supportcompanion.plist

```diff
@@ -278,6 +279,16 @@
 			<key>pfm_value_placeholder</key>
 			<string>111-222-333-4444</string>
 		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Configures whether to show the menu toggle button in the apps side menu.</string>
+			<key>pfm_name</key>
+			<string>ShowMenuToggle</string>
+			<key>pfm_title</key>
+			<string>Show Menu Toggle</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
```

Comment:
The interaction with boolean properties really benefits from them having default values (`pfm_default_value`). Do you mind adding it here?

Take this opportunity to add one to `IntuneMode` too if you can.

Reply:
I think I added what you suggested here :)